newIDE/app/src/ProjectManager/ProjectManagerItems.js

```diff
@@ -174,80 +181,103 @@ export const Item = ({
   return (
     <I18n>
       {({ i18n }) => (
-        <ListItem
-          id={id}
-          data={data}
-          style={
-            isLastItem
-              ? undefined
-              : {
-                  borderBottom: `1px solid ${
-                    gdevelopTheme.listItem.separatorColor
-                  }`,
-                }
-          }
-          noPadding
-          primaryText={label}
-          leftIcon={leftIcon}
-          displayMenuButton
-          buildMenuTemplate={(i18n: I18nType) => [
-            {
-              label: i18n._(t`Edit`),
-              click: onEdit,
-            },
-            ...(buildExtraMenuTemplate ? buildExtraMenuTemplate(i18n) : []),
-            { type: 'separator' },
-            {
-              label: i18n._(t`Rename`),
-              click: onEditName,
-            },
-            {
-              label: i18n._(t`Delete`),
-              click: onDelete,
-            },
-            {
-              label: i18n._(addLabel),
-              visible: !!onAdd,
-              click: onAdd,
-            },
-            { type: 'separator' },
-            {
-              label: i18n._(t`Copy`),
-              click: onCopy,
-            },
-            {
-              label: i18n._(t`Cut`),
-              click: onCut,
-            },
-            {
-              label: i18n._(t`Paste`),
-              enabled: canPaste(),
-              click: onPaste,
-            },
-            {
-              label: i18n._(t`Duplicate`),
-              click: onDuplicate,
-            },
-            { type: 'separator' },
-            {
-              label: i18n._(t`Move up`),
-              enabled: canMoveUp,
-              click: onMoveUp,
-            },
-            {
-              label: i18n._(t`Move down`),
-              enabled: canMoveDown,
-              click: onMoveDown,
-            },
-          ]}
-          onClick={() => {
-            // It's essential to discard clicks when editing the name,
-            // to avoid weird opening of an editor (accompanied with a
-            // closing of the project manager) when clicking on the text
-            // field.
-            if (!editingName) onEdit();
+        <DragSourceAndDropTarget
+          beginDrag={() => {
+            onBeginDrag();
+            return {};
           }}
-        />
+          canDrag={() => true}
+          canDrop={() => true}
+          drop={onDrop}
+        >
+          {({ connectDragSource, connectDropTarget, isOver, canDrop }) =>
+            connectDropTarget(
+              <div style={{ display: 'flex', flexDirection: 'column' }}>
```

Comment:
Might be worth putting into a `styles.itemDragDropContainer`?

Reply:
Oh yes I forgot, thanks